velox/type/DecimalUtil.h

```diff
@@ -85,6 +85,14 @@ class DecimalUtil {
         value);
   }
 
+  FOLLY_ALWAYS_INLINE static bool valueInRangeWithPrecision(
```

Comment:
Please use this API to check. https://github.com/facebookincubator/velox/pull/6384